src/modules/10-common/components/UserGroupsInput/UserGroupsInput.tsx

```diff
@@ -44,7 +44,7 @@ interface MappedUserGroupData {
 }
 
 export interface FormikUserGroupsInput extends UserGroupsInputProps {
-  formik: FormikContext<any>
+  formik: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
  formik: FormikContextType<unknown>
```